lib/explorer/series.ex

```diff
@@ -476,6 +476,20 @@ defmodule Explorer.Series do
   @spec last(series :: Series.t()) :: any()
   def last(series), do: Shared.apply_impl(series, :last, [])
 
+  @doc """
+  return a series from two series, based on a predicate.
+
+  The resulting series is built by evaluating each element of
+  `pred` and returning either the corresponding element from
```

Comment:
I think it's better if we use the full name: `predicate`. Same as the arguments for the functions.
It's common in our codebase to not use abbreviations.